lhotse/cut/mono.py

```diff
@@ -198,6 +198,11 @@ def num_samples(self) -> Optional[int]:
     def num_features(self) -> Optional[int]:
         return self.features.num_features if self.has_features else None
 
+    @property
+    def num_channels(self) -> Optional[int]:
+        # MonoCut is always single-channel, if it has a recording.
+        return int(self.has_recording)
```

Comment:
By definition MonoCut has a single channel regardless of the recording's presence, right?

```suggestion
        return 1
```

Reply:
But then what does this "ghost" channel refer to?

Reply:
If you have features instead, then they represent that single channel. If you have neither recording or features, then the object is sort of invalid anyway.

Reply:
Ahh, makes sense.